pubsub/tests/test_pubsub.go

```diff
@@ -154,6 +157,9 @@ type TestID string
 func NewTestID() TestID {
 	return TestID(watermill.NewUUID())
 }
+func NewTestID_ULID() TestID {
```

Comment:
```suggestion
func NewTestULID() TestID {
```
Just to keep the convention of no underscores in func names